commons/src/test/java/com/powsybl/commons/ComparisonUtils.java

```diff
@@ -0,0 +1,59 @@
+package com.powsybl.commons;
+
+import com.google.common.io.ByteStreams;
+import org.xmlunit.builder.DiffBuilder;
+import org.xmlunit.builder.Input;
+import org.xmlunit.diff.Diff;
+
+import javax.xml.transform.Source;
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+public class ComparisonUtils {
```

Comment:
This class should be final with a private constructor (see checkstyle issues for more details)